src/scratch/01_clean_api.r

```diff
@@ -548,7 +516,7 @@ options(cansim.cache_path = "C:\\Users\\JDUAN\\Downloads\\project\\data\\cansim_
 getOption("cansim.cache_path")
 connection <- cansim::get_cansim_sqlite(cansim_id,
                                         auto_refresh=TRUE
-                                        # refresh=TRUE # only occasionally refresh
+                                        # refresh=TRUE # only occationally refresh
```

Comment:
it looks like it undid this typo fix with the merge

Reply:
Weird. I changed it now.